browser/src/layer/tile/CommentListSection.ts

```diff
@@ -1062,6 +1068,18 @@ export class CommentSection extends CanvasSectionObject {
 		return index == -1 ? null : this.sectionProperties.commentList[index];
 	}
 
+	private checkIfCommentHasPreAssignedChildren(comment: CommentSection) {
```

Comment:
name could be better, it doesn't check anything (which would be getter/readonly operation)  but rather adds children to the old parent (does modification, which coudn't be guessed from the name)